src/pages/home/components/Sidebar/Sidebar.tsx

```diff
@@ -1,70 +1,96 @@
 import { useRef, useState } from 'react';
 
+import classNames from 'classnames';
+
 import AccordionTab from '~components/AccordionTab/AccordionTab';
 import Button from '~components/Button/Button';
 import InputSearch from '~components/InputSearch/InputSearch';
 import { TInputComponent } from '~components/InputSearch/InputSearch.types';
 import Modal from '~components/Modal/Modal';
 import SearchClue from '~components/SearchClue/SearchClue';
 
+import { PlusIcon } from './components/PlusIcon';
+
 import scss from './Sidebar.module.scss';
 
 function Sidebar() {
   const [value, setValue] = useState('');
   const [isOpen, setOpen] = useState(false);
+  const [mobileIsOpen, setMobileIsOpen] = useState(false);
   const inputSearchRef = useRef<TInputComponent | null>(null);
 
   const handleToggleModal = () => {
     setOpen((isOpen) => !isOpen);
   };
 
-  const renderSearchClue = () => {
-    return (
-      <SearchClue
-        clearInput={inputSearchRef.current?.clearInput}
-        value={value}
-        label="Searching for"
-      />
-    );
-  };
-
-  return (
-    <AccordionTab title="Select Social Media" hideCloseButton>
-      <div className={scss.content}>
-        <InputSearch
-          className={scss.searchBar}
-          onChange={(value) => setValue(value as string)}
-          placeholder="Search for social media"
-          ref={inputSearchRef}
-          error={false}
-        />
+  const renderSearchClue = () => (
+    <SearchClue
+      clearInput={inputSearchRef.current?.clearInput}
+      label="Searching for"
+      value={value}
+    />
+  );
 
-        {!value && renderSearchClue()}
+  const mobalIsOpen = mobileIsOpen ? scss.openMobile : scss.closeMobile;
 
-        <div className={scss.items}>
-          Item 1 <br /> Item2 <br /> Item 1 <br /> Item2 <br />
-          Item 1 <br /> Item2 <br /> Item 1 <br /> Item2 <br />
-          Item 1 <br /> Item2 <br /> Item 1 <br /> Item2 <br />
-          Item 1 <br /> Item2 <br /> Item 1 <br /> Item2 <br />
-          Item 1 <br /> Item2 <br /> Item 1 <br /> Item2 <br />
-          Item 1 <br /> Item2 <br /> Item 1 <br /> Item2 <br />
-          Item 1 <br /> Item2 <br /> Item 1 <br /> Item2 <br />
-          Item 1 <br /> Item2 <br /> Item 1 <br /> Item2 <br />
+  return (
+    <div>
+      <button onClick={() => setMobileIsOpen(!mobileIsOpen)}>Open/Close</button>
```

Comment:
Essa abordagem é perigosa e pode levar a alguns leaks no estado.

https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state

Se vai usar o estado atual para setar o estado futuro, sempre prefira uma `updater function`.
```tsx
<button onClick={() => setMobileIsOpen((currentMobileIsOpen) => !currentMobileIsOpen)}>Open/Close</button>
```